src/hotspot/share/runtime/os.hpp

```diff
@@ -763,6 +763,7 @@ class os: AllStatic {
   static void print_siginfo(outputStream* st, const void* siginfo);
   static void print_signal_handlers(outputStream* st, char* buf, size_t buflen);
   static void print_date_and_time(outputStream* st, char* buf, size_t buflen);
+  static void print_stack(outputStream* st, address sp, int unitsize);
```

Comment:
I'd rename this to something like `print_stack_as_hex_dump()` or similar since print_stack suggests you print a stack, and because we already have `platform_print_native_stack()` and `get_native_stack()`.

Reply:
print_tos_pc itself is weird, since it does two unrelated things, but this is fine for a separate cleanup. Thanks for renaming.

Reply:
Done in new commit.